packages/celltags/src/addwidget.ts

```diff
@@ -9,8 +14,10 @@ export class AddWidget extends Widget {
   /**
    * Construct a new tag widget.
    */
-  constructor() {
+  constructor(translator: ITranslator) {
```

Comment:
optional?

Reply:
Fixed. (going over a search of all translators...)